ceph/utils.py

```diff
@@ -506,3 +512,63 @@ def get_public_network():
         (str) public network subnet
     """
     return "10.0.144.0/22"  # TODO: pull from configuration file
+
+
+@retry(LibcloudError, tries=5, delay=15)
```

Comment:
save these as constants in a config file

Reply:
part of backlog since its a code refactor and not new code.